sdk/core/System.ClientModel/tests/Samples/Readme.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Text.Json;
+
+namespace System.ClientModel.Tests.Samples
+{
+    internal class Readme
```

Comment:
Please name this file with a name related to ModelReaderWriter, following the precedents in Azure.Core generated snippets.

Reply:
Why not follow Azure.Core naming conventions?